sklearn/linear_model/tests/test_bayes.py

```diff
@@ -94,6 +94,17 @@ def test_bayesian_ridge_parameter():
     assert_almost_equal(rr_model.intercept_, br_model.intercept_)
 
 
+def test_bayesian_covariance_matrix():
+    # Check the posterior covariance matrix sigma_
+    X, y = datasets.make_regression(n_samples=10, n_features=20)
```

Comment:
Maybe parametrize `"n_samples, n_features", [(10, 20), (20, 10)]` to check that both regimes lead to consistent results?